modules/virt-troubleshooting-vm-import.adoc

```diff
@@ -0,0 +1,110 @@
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the V2V Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+You can check the VM Import Controller Pod log for errors.
+
+.Procedure
+
+. Obtain the VM Import Controller Pod name:
++
+----
+$ oc get pods -n <project> | grep import <1>
+vm-import-controller-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the VM Import Controller Pod log:
++
+----
+$ oc logs vm-import-controller-f66f7d-zqkz7 -f -n <project>
+----
+endif::[]
+
+[id='error-messages_{context}']
+== Error messages
+
+The following error messages might appear:
+
+ifdef::virt-importing-rhv-vm[]
+* The following error message is displayed in the VM Import Controller Pod log if the RHV VM's system settings do not emulate the Intel Q35 chipset:
++
+----
+The virtual machine could not be imported.
+MappingRulesVerificationFailed: VM uses unsupported bios type: i440fx_sea_bios
+----
+
+* The following error message is displayed in the VM Import Controller Pod log if the target VM name exceeds 23 characters link:https://bugzilla.redhat.com/show_bug.cgi?id=1857165[(*BZ#1857165*)]:
++
+----
+Message:               Error while importing disk image
+Reason:                ProcessingFailed
+----
+
+* The following error message is displayed in the VM Import Controller Pod log and the progress bar stops at 10% if the {VirtProductName} storage PV is not suitable:
++
+----
+Failed to bind volumes: provisioning failed for PVC
+----
++
+You must use Ceph Filesystem or local storage. Cinder storage is not supported. link:https://bugzilla.redhat.com/show_bug.cgi?id=1857784[(*BZ#1857784*)]
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+* If the VMware VM is not shut down before import, the imported virtual machine displays the error message, `Readiness probe failed` in the {product-title} console and the V2V Conversion Pod log displays the following error message:
++
+----
+INFO - have error: ('virt-v2v error: internal error: invalid argument: libvirt domain ‘v2v_migration_vm_1’ is running or paused. It must be shut down in order to perform virt-v2v conversion',)"
+----
+endif::[]
+
+[id='known-issues_{context}']
+== Known issues
+
+The following are known issues:
+
+ifdef::virt-importing-vmware-vm[]
+* You must have sufficient storage space for the imported disk.
++
+If you try to import a virtual machine whose disk size is larger than the available storage space, the operation cannot complete. You will not be able to import another virtual machine or to clean up the storage because there are insufficient resources to support object deletion. To resolve this situation, you must add more object storage devices to the storage backend. link:https://bugzilla.redhat.com/show_bug.cgi?id=1721504[(*BZ#1721504*)]
```

Comment:
I think "whose" personifies the VM too much :)

Reply:
Wow. Never thought of 'whose' as an anthropomorphism. OK, will rephrase.